tests/storage/storage.go

```diff
@@ -158,18 +167,12 @@ var _ = SIGDescribe("Storage", func() {
 
 				refresh := ThisVMI(vmi)
 				By("Expecting VMI to be paused")
-				Eventually(func() bool {
+				Eventually(func() *v1.VirtualMachineInstance {
 					vmi, err := refresh()
 					Expect(err).ToNot(HaveOccurred())
 
-					for _, condition := range vmi.Status.Conditions {
-						if condition.Type == virtv1.VirtualMachineInstancePaused {
-							return condition.Status == k8sv1.ConditionTrue && condition.Reason == "PausedIOError"
-						}
-					}
-					return false
-
-				}, 100*time.Second, time.Second).Should(BeTrue())
+					return vmi
```

Comment:
maybe instead of returning vmi return vmi.Status.Conditions? so if it fails the print will be only that and not all the vmi?

Reply:
you are right, done

Reply:
cool